source/common/runtime/runtime_features.cc

```diff
@@ -74,6 +74,7 @@ constexpr const char* runtime_features[] = {
     "envoy.reloadable_features.http_transport_failure_reason_in_body",
     "envoy.reloadable_features.http_upstream_wait_connect_response",
     "envoy.reloadable_features.http2_skip_encoding_empty_trailers",
+    "envoy.reloadable_features.legacy_sotw_xds",
```

Comment:
Can you give some details on the control plane facing changes that enabling/disabling this flag causes?

Reply:
closed in favour of https://github.com/envoyproxy/envoy/pull/15473